<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="147" failures="22" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
11:14:38 INFO  Dir:   plugin_apama/
11:14:38 INFO  Title: Validate apama plugin install
11:14:38 INFO  ==============================================================
11:14:38 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.05 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
11:14:38 INFO  Dir:   plugin_apama/
11:14:38 INFO  Title: Validate mon file support of apama plugin
11:14:38 INFO  ==============================================================
11:14:38 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.03 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
11:14:38 INFO  Dir:   plugin_apama/
11:14:38 INFO  Title: Validate apama plugin remove
11:14:38 INFO  ==============================================================
11:14:38 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.02 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
11:14:38 INFO  Dir:   plugin_apama/
11:14:38 INFO  Title: Validate apama plugin updating one project with another
11:14:38 INFO  ==============================================================
11:14:38 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.02 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>12:05:53 INFO  ==============================================================
12:05:53 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
12:05:53 INFO  Dir:   plugin_apama/
12:05:53 INFO  Title: Validate apama plugin updating one project with another
12:05:53 INFO  ==============================================================
12:05:53 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
12:05:53 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
12:05:54 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
12:05:54 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
12:05:55 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
12:05:56 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
12:05:56 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
12:05:57 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
12:05:57 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
12:05:57 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
12:05:57 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

12:05:58 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
12:05:58 WARN  Aborted test due to BLOCKED outcome
12:05:58 INFO  
12:05:58 INFO  cleanup:
12:05:58 INFO  Removing apama project
12:06:13 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
12:06:13 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
12:06:14 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
12:06:14 INFO  
12:06:14 INFO  Test duration: 21.31 secs
12:06:14 INFO  Test final outcome:  BLOCKED
12:06:14 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
12:06:14 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>12:10:10 INFO  ==============================================================
12:10:10 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
12:10:10 INFO  Dir:   plugin_apama/
12:10:10 INFO  Title: Validate apama plugin updating one project with another
12:10:10 INFO  ==============================================================
12:10:10 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
12:10:10 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
12:10:11 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
12:10:12 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
12:10:12 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
12:10:13 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
12:10:13 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
12:10:14 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
12:10:14 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
12:10:14 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
12:10:15 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

12:10:15 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
12:10:15 WARN  Aborted test due to BLOCKED outcome
12:10:15 INFO  
12:10:15 INFO  cleanup:
12:10:15 INFO  Removing apama project
12:10:30 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
12:10:30 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
12:10:31 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
12:10:31 INFO  
12:10:31 INFO  Test duration: 21.01 secs
12:10:31 INFO  Test final outcome:  BLOCKED
12:10:31 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
12:10:31 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Install package to directory specified in tedge config set tmp.path
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.06 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>11:47:52 INFO  ==============================================================
11:47:52 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
11:47:52 INFO  Dir:   software_management_end_to_end/
11:47:52 INFO  Title: Install package to directory specified in tedge config set tmp.path
11:47:52 INFO  ==============================================================
11:47:52 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:47:52 INFO  EnvironmentC8y Setup
11:47:53 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:48:07 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:48:09 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:48:10 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:48:11 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:48:13 INFO  State of current operation: SUCCESSFUL
11:48:13 INFO  Assert that (False == value) with value=False ... passed
11:48:14 INFO  Executed sudo, exit status 0
11:48:14 INFO  Executed sudo, exit status 0
11:48:15 INFO  Executed sudo, exit status 0
11:48:19 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:48:19 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
11:48:19 INFO  Delaying execution by 7 seconds
11:48:40 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:48:40 INFO  Response status: 201
11:48:40 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-05-20T11:48:40.725Z",
        "status": "PENDING"
    },
    "creationTime": "2022-05-20T11:48:40.709Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409225",
    "id": "19409225",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
11:48:40 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-05-20T11:48:40.725Z', 'status': 'PENDING'}, 'creationTime': '2022-05-20T11:48:40.709Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409225', 'id': '19409225', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
11:48:40 WARN  , retrying in 0.5 seconds...
11:48:41 WARN  , retrying in 0.5 seconds...
11:48:41 WARN  , retrying in 0.5 seconds...
11:48:42 WARN  , retrying in 0.5 seconds...
11:48:42 WARN  , retrying in 0.5 seconds...
11:48:43 WARN  , retrying in 0.5 seconds...
11:48:43 WARN  , retrying in 0.5 seconds...
11:48:44 WARN  , retrying in 0.5 seconds...
11:48:44 WARN  , retrying in 0.5 seconds...
11:48:45 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
11:48:45 WARN  AssertionError: ... blocked 
11:48:45 INFO  
11:48:45 INFO  cleanup:
11:48:46 INFO  Executed sudo, exit status 0
11:48:47 INFO  Executed sudo, exit status 0
11:48:51 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:48:51 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:48:51 INFO  Delaying execution by 8 seconds
11:49:13 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:49:17 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:49:18 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:49:18 INFO  
11:49:18 INFO  Test duration: 85.93 secs
11:49:18 INFO  Test final outcome:  BLOCKED
11:49:18 INFO  Test outcome reason: AssertionError:
11:49:18 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>12:10:31 INFO  ==============================================================
12:10:31 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
12:10:31 INFO  Dir:   software_management_end_to_end/
12:10:31 INFO  Title: Install package to directory specified in tedge config set tmp.path
12:10:31 INFO  ==============================================================
12:10:32 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:10:32 INFO  EnvironmentC8y Setup
12:10:33 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:10:47 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:10:47 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:10:48 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:10:48 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:10:51 INFO  State of current operation: FAILED
12:10:51 INFO  Assert that (False == value) with value=False ... passed
12:10:51 INFO  Executed sudo, exit status 0
12:10:52 INFO  Executed sudo, exit status 0
12:10:53 INFO  Executed sudo, exit status 0
12:10:57 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:10:57 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
12:10:57 INFO  Delaying execution by 8 seconds
12:11:19 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:11:19 INFO  Response status: 201
12:11:19 INFO  Response to action: {
    "creationTime": "2022-05-20T12:11:19.455Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409391",
    "id": "19409391",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
12:11:19 INFO  Started operation: {'creationTime': '2022-05-20T12:11:19.455Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409391', 'id': '19409391', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
12:11:19 WARN  , retrying in 0.5 seconds...
12:11:20 WARN  , retrying in 0.5 seconds...
12:11:20 WARN  , retrying in 0.5 seconds...
12:11:21 WARN  , retrying in 0.5 seconds...
12:11:21 WARN  , retrying in 0.5 seconds...
12:11:22 WARN  , retrying in 0.5 seconds...
12:11:22 WARN  , retrying in 0.5 seconds...
12:11:23 WARN  , retrying in 0.5 seconds...
12:11:23 WARN  , retrying in 0.5 seconds...
12:11:24 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
12:11:24 WARN  AssertionError: ... blocked 
12:11:24 INFO  
12:11:24 INFO  cleanup:
12:11:24 INFO  Executed sudo, exit status 0
12:11:25 INFO  Executed sudo, exit status 0
12:11:29 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:11:29 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
12:11:29 INFO  Delaying execution by 7 seconds
12:11:51 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:11:55 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:11:55 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:11:55 INFO  
12:11:55 INFO  Test duration: 84.36 secs
12:11:55 INFO  Test final outcome:  BLOCKED
12:11:55 INFO  Test outcome reason: AssertionError:
12:11:55 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_apt_install_fail
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for a failing installation
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.02 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_apt_install_remove
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.02 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.02 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.02 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>11:56:05 INFO  ==============================================================
11:56:05 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
11:56:05 INFO  Dir:   software_management_end_to_end/
11:56:05 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
11:56:05 INFO  ==============================================================
11:56:05 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:56:05 INFO  EnvironmentC8y Setup
11:56:06 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:56:06 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:56:06 INFO  Delaying execution by 6 seconds
11:56:26 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:56:27 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:56:27 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:56:28 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:56:30 INFO  State of current operation: SUCCESSFUL
11:56:30 INFO  Assert that (False == value) with value=False ... passed
11:56:30 INFO  Assert that (False == value) with value=False ... passed
11:56:31 INFO  Assert that (False == value) with value=False ... passed
11:56:31 INFO  Package robotfindskitten is installed
11:56:31 WARN  Assert that (False == value) with value=True ... failed 
11:56:31 INFO  Response status: 201
11:56:31 INFO  Response to action: {
    "creationTime": "2022-05-20T11:56:31.626Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409300",
    "id": "19409300",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:56:31 INFO  Started operation: {'creationTime': '2022-05-20T11:56:31.626Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409300', 'id': '19409300', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:56:33 INFO  State of operation 19409300 : PENDING
11:56:36 INFO  State of operation 19409300 : EXECUTING
11:56:38 INFO  State of operation 19409300 : EXECUTING
11:56:40 INFO  State of operation 19409300 : EXECUTING
11:56:42 INFO  State of operation 19409300 : EXECUTING
11:56:45 INFO  State of operation 19409300 : EXECUTING
11:56:48 INFO  State of operation 19409300 : EXECUTING
11:56:50 INFO  State of operation 19409300 : EXECUTING
11:56:53 INFO  State of operation 19409300 : EXECUTING
11:56:55 INFO  State of operation 19409300 : EXECUTING
11:56:57 INFO  State of operation 19409300 : EXECUTING
11:56:59 INFO  State of operation 19409300 : EXECUTING
11:57:02 INFO  State of operation 19409300 : EXECUTING
11:57:04 INFO  State of operation 19409300 : EXECUTING
11:57:06 INFO  State of operation 19409300 : EXECUTING
11:57:08 INFO  State of operation 19409300 : EXECUTING
11:57:11 INFO  State of operation 19409300 : EXECUTING
11:57:14 INFO  State of operation 19409300 : EXECUTING
11:57:16 INFO  State of operation 19409300 : EXECUTING
11:57:18 INFO  State of operation 19409300 : EXECUTING
11:57:20 INFO  State of operation 19409300 : EXECUTING
11:57:23 INFO  State of operation 19409300 : EXECUTING
11:57:25 INFO  State of operation 19409300 : EXECUTING
11:57:28 INFO  State of operation 19409300 : EXECUTING
11:57:30 INFO  State of operation 19409300 : EXECUTING
11:57:33 INFO  State of operation 19409300 : EXECUTING
11:57:35 INFO  State of operation 19409300 : EXECUTING
11:57:38 INFO  State of operation 19409300 : EXECUTING
11:57:40 INFO  State of operation 19409300 : EXECUTING
11:57:43 INFO  State of operation 19409300 : EXECUTING
11:57:45 INFO  State of operation 19409300 : EXECUTING
11:57:47 INFO  State of operation 19409300 : EXECUTING
11:57:49 INFO  State of operation 19409300 : EXECUTING
11:57:52 INFO  State of operation 19409300 : SUCCESSFUL
11:57:52 INFO  Assert that (False == value) with value=False ... passed
11:57:52 INFO  Package asciijump is installed
11:57:52 INFO  Assert that (True == value) with value=True ... passed
11:57:52 INFO  Package squirrel3 is installed
11:57:52 INFO  Assert that (True == value) with value=True ... passed
11:57:52 INFO  Package robotfindskitten is installed
11:57:52 INFO  Assert that (True == value) with value=True ... passed
11:57:53 INFO  Response status: 201
11:57:53 INFO  Response to action: {
    "creationTime": "2022-05-20T11:57:53.132Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409314",
    "id": "19409314",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:57:53 INFO  Started operation: {'creationTime': '2022-05-20T11:57:53.132Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409314', 'id': '19409314', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:57:55 INFO  State of operation 19409314 : PENDING
11:57:57 INFO  State of operation 19409314 : EXECUTING
11:57:59 INFO  State of operation 19409314 : EXECUTING
11:58:02 INFO  State of operation 19409314 : EXECUTING
11:58:04 INFO  State of operation 19409314 : EXECUTING
11:58:06 INFO  State of operation 19409314 : EXECUTING
11:58:09 INFO  State of operation 19409314 : EXECUTING
11:58:12 INFO  State of operation 19409314 : EXECUTING
11:58:14 INFO  State of operation 19409314 : EXECUTING
11:58:16 INFO  State of operation 19409314 : EXECUTING
11:58:19 INFO  State of operation 19409314 : EXECUTING
11:58:21 INFO  State of operation 19409314 : EXECUTING
11:58:23 INFO  State of operation 19409314 : EXECUTING
11:58:25 INFO  State of operation 19409314 : EXECUTING
11:58:28 INFO  State of operation 19409314 : EXECUTING
11:58:30 INFO  State of operation 19409314 : EXECUTING
11:58:32 INFO  State of operation 19409314 : EXECUTING
11:58:35 INFO  State of operation 19409314 : EXECUTING
11:58:38 INFO  State of operation 19409314 : EXECUTING
11:58:40 INFO  State of operation 19409314 : EXECUTING
11:58:42 INFO  State of operation 19409314 : EXECUTING
11:58:44 INFO  State of operation 19409314 : EXECUTING
11:58:46 INFO  State of operation 19409314 : EXECUTING
11:58:49 INFO  State of operation 19409314 : EXECUTING
11:58:51 INFO  State of operation 19409314 : EXECUTING
11:58:54 INFO  State of operation 19409314 : EXECUTING
11:58:56 INFO  State of operation 19409314 : EXECUTING
11:58:58 INFO  State of operation 19409314 : SUCCESSFUL
11:58:58 INFO  Assert that (False == value) with value=False ... passed
11:58:59 INFO  Assert that (False == value) with value=False ... passed
11:58:59 INFO  Assert that (False == value) with value=False ... passed
11:58:59 INFO  Assert that (False == value) with value=False ... passed
11:58:59 INFO  
11:58:59 INFO  cleanup:
11:59:03 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:59:04 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:59:04 INFO  
11:59:04 INFO  Test duration: 178.33 secs
11:59:04 INFO  Test final outcome:  FAILED
11:59:04 INFO  Test outcome reason: Assert that (False == value) with value=True
11:59:04 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>12:18:18 INFO  ==============================================================
12:18:18 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
12:18:18 INFO  Dir:   software_management_end_to_end/
12:18:18 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
12:18:18 INFO  ==============================================================
12:18:18 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:18:18 INFO  EnvironmentC8y Setup
12:18:18 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:18:18 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
12:18:18 INFO  Delaying execution by 6 seconds
12:18:38 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:18:39 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:18:39 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:18:40 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:18:42 INFO  State of current operation: SUCCESSFUL
12:18:42 INFO  Assert that (False == value) with value=False ... passed
12:18:43 INFO  Package asciijump is installed
12:18:43 WARN  Assert that (False == value) with value=True ... failed 
12:18:43 INFO  Package squirrel3 is installed
12:18:43 WARN  Assert that (False == value) with value=True ... failed 
12:18:43 INFO  Package robotfindskitten is installed
12:18:43 WARN  Assert that (False == value) with value=True ... failed 
12:18:44 INFO  Response status: 201
12:18:44 INFO  Response to action: {
    "creationTime": "2022-05-20T12:18:44.080Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409477",
    "id": "19409477",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
12:18:44 INFO  Started operation: {'creationTime': '2022-05-20T12:18:44.080Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409477', 'id': '19409477', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
12:18:46 INFO  State of operation 19409477 : PENDING
12:18:48 INFO  State of operation 19409477 : EXECUTING
12:18:50 INFO  State of operation 19409477 : EXECUTING
12:18:53 INFO  State of operation 19409477 : EXECUTING
12:18:55 INFO  State of operation 19409477 : EXECUTING
12:18:57 INFO  State of operation 19409477 : EXECUTING
12:19:00 INFO  State of operation 19409477 : EXECUTING
12:19:03 INFO  State of operation 19409477 : EXECUTING
12:19:05 INFO  State of operation 19409477 : EXECUTING
12:19:08 INFO  State of operation 19409477 : EXECUTING
12:19:10 INFO  State of operation 19409477 : EXECUTING
12:19:12 INFO  State of operation 19409477 : EXECUTING
12:19:14 INFO  State of operation 19409477 : EXECUTING
12:19:17 INFO  State of operation 19409477 : EXECUTING
12:19:19 INFO  State of operation 19409477 : EXECUTING
12:19:21 INFO  State of operation 19409477 : SUCCESSFUL
12:19:21 INFO  Assert that (False == value) with value=False ... passed
12:19:22 INFO  Package asciijump is installed
12:19:22 INFO  Assert that (True == value) with value=True ... passed
12:19:22 INFO  Package squirrel3 is installed
12:19:22 INFO  Assert that (True == value) with value=True ... passed
12:19:22 INFO  Package robotfindskitten is installed
12:19:22 INFO  Assert that (True == value) with value=True ... passed
12:19:22 INFO  Response status: 201
12:19:22 INFO  Response to action: {
    "creationTime": "2022-05-20T12:19:22.692Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409486",
    "id": "19409486",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
12:19:22 INFO  Started operation: {'creationTime': '2022-05-20T12:19:22.692Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409486', 'id': '19409486', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
12:19:24 INFO  State of operation 19409486 : PENDING
12:19:27 INFO  State of operation 19409486 : EXECUTING
12:19:29 INFO  State of operation 19409486 : EXECUTING
12:19:31 INFO  State of operation 19409486 : EXECUTING
12:19:33 INFO  State of operation 19409486 : EXECUTING
12:19:36 INFO  State of operation 19409486 : EXECUTING
12:19:38 INFO  State of operation 19409486 : EXECUTING
12:19:41 INFO  State of operation 19409486 : EXECUTING
12:19:43 INFO  State of operation 19409486 : EXECUTING
12:19:46 INFO  State of operation 19409486 : EXECUTING
12:19:48 INFO  State of operation 19409486 : EXECUTING
12:19:50 INFO  State of operation 19409486 : EXECUTING
12:19:52 INFO  State of operation 19409486 : EXECUTING
12:19:55 INFO  State of operation 19409486 : EXECUTING
12:19:57 INFO  State of operation 19409486 : EXECUTING
12:19:59 INFO  State of operation 19409486 : EXECUTING
12:20:01 INFO  State of operation 19409486 : EXECUTING
12:20:04 INFO  State of operation 19409486 : EXECUTING
12:20:07 INFO  State of operation 19409486 : EXECUTING
12:20:09 INFO  State of operation 19409486 : EXECUTING
12:20:11 INFO  State of operation 19409486 : EXECUTING
12:20:13 INFO  State of operation 19409486 : EXECUTING
12:20:15 INFO  State of operation 19409486 : EXECUTING
12:20:18 INFO  State of operation 19409486 : EXECUTING
12:20:20 INFO  State of operation 19409486 : EXECUTING
12:20:23 INFO  State of operation 19409486 : EXECUTING
12:20:25 INFO  State of operation 19409486 : EXECUTING
12:20:27 INFO  State of operation 19409486 : EXECUTING
12:20:29 INFO  State of operation 19409486 : SUCCESSFUL
12:20:30 INFO  Assert that (False == value) with value=False ... passed
12:20:30 INFO  Assert that (False == value) with value=False ... passed
12:20:30 INFO  Assert that (False == value) with value=False ... passed
12:20:30 INFO  Assert that (False == value) with value=False ... passed
12:20:30 INFO  
12:20:30 INFO  cleanup:
12:20:35 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:20:35 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:20:35 INFO  
12:20:35 INFO  Test duration: 137.76 secs
12:20:35 INFO  Test final outcome:  FAILED
12:20:35 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
12:20:35 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.03 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>11:59:04 INFO  ==============================================================
11:59:04 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
11:59:04 INFO  Dir:   software_management_end_to_end/
11:59:04 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
11:59:04 INFO  ==============================================================
11:59:04 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:59:04 INFO  EnvironmentC8y Setup
11:59:04 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:59:05 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:59:05 INFO  Delaying execution by 6 seconds
11:59:24 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:59:25 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:59:25 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:59:26 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:59:28 INFO  State of current operation: SUCCESSFUL
11:59:28 INFO  Assert that (False == value) with value=False ... passed
11:59:29 INFO  Assert that (False == value) with value=False ... passed
11:59:29 INFO  Assert that (False == value) with value=False ... passed
11:59:29 INFO  Assert that (False == value) with value=False ... passed
11:59:29 INFO  Assert that (False == value) with value=False ... passed
11:59:31 INFO  Response status: 201
11:59:31 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-05-20T11:59:31.016Z",
        "status": "PENDING"
    },
    "creationTime": "2022-05-20T11:59:31.000Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409327",
    "id": "19409327",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:59:31 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-05-20T11:59:31.016Z', 'status': 'PENDING'}, 'creationTime': '2022-05-20T11:59:31.000Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409327', 'id': '19409327', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
11:59:33 INFO  State of operation 19409327 : PENDING
11:59:35 INFO  State of operation 19409327 : EXECUTING
11:59:37 INFO  State of operation 19409327 : EXECUTING
11:59:40 INFO  State of operation 19409327 : EXECUTING
11:59:42 INFO  State of operation 19409327 : EXECUTING
11:59:44 INFO  State of operation 19409327 : EXECUTING
11:59:47 INFO  State of operation 19409327 : EXECUTING
11:59:50 INFO  State of operation 19409327 : EXECUTING
11:59:52 INFO  State of operation 19409327 : EXECUTING
11:59:54 INFO  State of operation 19409327 : EXECUTING
11:59:57 INFO  State of operation 19409327 : EXECUTING
11:59:59 INFO  State of operation 19409327 : EXECUTING
12:00:01 INFO  State of operation 19409327 : EXECUTING
12:00:03 INFO  State of operation 19409327 : EXECUTING
12:00:06 INFO  State of operation 19409327 : EXECUTING
12:00:08 INFO  State of operation 19409327 : EXECUTING
12:00:10 INFO  State of operation 19409327 : EXECUTING
12:00:13 INFO  State of operation 19409327 : EXECUTING
12:00:15 INFO  State of operation 19409327 : EXECUTING
12:00:18 INFO  State of operation 19409327 : EXECUTING
12:00:20 INFO  State of operation 19409327 : EXECUTING
12:00:23 INFO  State of operation 19409327 : EXECUTING
12:00:25 INFO  State of operation 19409327 : EXECUTING
12:00:28 INFO  State of operation 19409327 : EXECUTING
12:00:30 INFO  State of operation 19409327 : EXECUTING
12:00:33 INFO  State of operation 19409327 : EXECUTING
12:00:35 INFO  State of operation 19409327 : EXECUTING
12:00:38 INFO  State of operation 19409327 : EXECUTING
12:00:40 INFO  State of operation 19409327 : EXECUTING
12:00:42 INFO  State of operation 19409327 : EXECUTING
12:00:44 INFO  State of operation 19409327 : EXECUTING
12:00:47 INFO  State of operation 19409327 : EXECUTING
12:00:49 INFO  State of operation 19409327 : EXECUTING
12:00:51 INFO  State of operation 19409327 : EXECUTING
12:00:53 INFO  State of operation 19409327 : EXECUTING
12:00:56 INFO  State of operation 19409327 : EXECUTING
12:00:58 INFO  State of operation 19409327 : EXECUTING
12:01:00 INFO  State of operation 19409327 : EXECUTING
12:01:03 INFO  State of operation 19409327 : EXECUTING
12:01:06 INFO  State of operation 19409327 : EXECUTING
12:01:08 INFO  State of operation 19409327 : EXECUTING
12:01:10 INFO  State of operation 19409327 : EXECUTING
12:01:12 INFO  State of operation 19409327 : EXECUTING
12:01:15 INFO  State of operation 19409327 : EXECUTING
12:01:17 INFO  State of operation 19409327 : EXECUTING
12:01:19 INFO  State of operation 19409327 : EXECUTING
12:01:21 INFO  State of operation 19409327 : EXECUTING
12:01:24 INFO  State of operation 19409327 : EXECUTING
12:01:26 INFO  State of operation 19409327 : EXECUTING
12:01:29 INFO  State of operation 19409327 : EXECUTING
12:01:31 INFO  State of operation 19409327 : EXECUTING
12:01:33 INFO  State of operation 19409327 : EXECUTING
12:01:37 INFO  State of operation 19409327 : EXECUTING
12:01:39 INFO  State of operation 19409327 : EXECUTING
12:01:42 INFO  State of operation 19409327 : EXECUTING
12:01:44 INFO  State of operation 19409327 : EXECUTING
12:01:46 INFO  State of operation 19409327 : EXECUTING
12:01:49 INFO  State of operation 19409327 : EXECUTING
12:01:51 INFO  State of operation 19409327 : EXECUTING
12:01:53 INFO  State of operation 19409327 : EXECUTING
12:01:56 INFO  State of operation 19409327 : EXECUTING
12:01:58 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
12:01:58 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
12:01:58 INFO  
12:01:58 INFO  cleanup:
12:02:02 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:02:03 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:02:03 INFO  
12:02:03 INFO  Test duration: 179.05 secs
12:02:03 INFO  Test final outcome:  BLOCKED
12:02:03 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
12:02:03 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>12:20:35 INFO  ==============================================================
12:20:35 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
12:20:35 INFO  Dir:   software_management_end_to_end/
12:20:35 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
12:20:35 INFO  ==============================================================
12:20:35 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:20:35 INFO  EnvironmentC8y Setup
12:20:36 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:20:36 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
12:20:36 INFO  Delaying execution by 6 seconds
12:20:56 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:20:57 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:20:57 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:20:58 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:21:00 INFO  State of current operation: SUCCESSFUL
12:21:00 INFO  Assert that (False == value) with value=False ... passed
12:21:00 INFO  Assert that (False == value) with value=False ... passed
12:21:01 INFO  Assert that (False == value) with value=False ... passed
12:21:01 INFO  Assert that (False == value) with value=False ... passed
12:21:01 INFO  Package moon-buggy is installed
12:21:01 WARN  Assert that (False == value) with value=True ... failed 
12:21:02 INFO  Response status: 201
12:21:02 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-05-20T12:21:02.819Z",
        "status": "PENDING"
    },
    "creationTime": "2022-05-20T12:21:02.800Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409494",
    "id": "19409494",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
12:21:02 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-05-20T12:21:02.819Z', 'status': 'PENDING'}, 'creationTime': '2022-05-20T12:21:02.800Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409494', 'id': '19409494', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
12:21:05 INFO  State of operation 19409494 : PENDING
12:21:07 INFO  State of operation 19409494 : EXECUTING
12:21:09 INFO  State of operation 19409494 : EXECUTING
12:21:11 INFO  State of operation 19409494 : EXECUTING
12:21:14 INFO  State of operation 19409494 : EXECUTING
12:21:16 INFO  State of operation 19409494 : EXECUTING
12:21:18 INFO  State of operation 19409494 : EXECUTING
12:21:21 INFO  State of operation 19409494 : EXECUTING
12:21:23 INFO  State of operation 19409494 : EXECUTING
12:21:26 INFO  State of operation 19409494 : EXECUTING
12:21:28 INFO  State of operation 19409494 : EXECUTING
12:21:30 INFO  State of operation 19409494 : EXECUTING
12:21:33 INFO  State of operation 19409494 : EXECUTING
12:21:35 INFO  State of operation 19409494 : EXECUTING
12:21:37 INFO  State of operation 19409494 : EXECUTING
12:21:39 INFO  State of operation 19409494 : EXECUTING
12:21:42 INFO  State of operation 19409494 : EXECUTING
12:21:44 INFO  State of operation 19409494 : EXECUTING
12:21:47 INFO  State of operation 19409494 : EXECUTING
12:21:49 INFO  State of operation 19409494 : EXECUTING
12:21:51 INFO  State of operation 19409494 : EXECUTING
12:21:53 INFO  State of operation 19409494 : EXECUTING
12:21:56 INFO  State of operation 19409494 : EXECUTING
12:21:58 INFO  State of operation 19409494 : EXECUTING
12:22:01 INFO  State of operation 19409494 : EXECUTING
12:22:03 INFO  State of operation 19409494 : EXECUTING
12:22:06 INFO  State of operation 19409494 : EXECUTING
12:22:08 INFO  State of operation 19409494 : EXECUTING
12:22:10 INFO  State of operation 19409494 : EXECUTING
12:22:12 INFO  State of operation 19409494 : EXECUTING
12:22:15 INFO  State of operation 19409494 : EXECUTING
12:22:17 INFO  State of operation 19409494 : EXECUTING
12:22:19 INFO  State of operation 19409494 : EXECUTING
12:22:21 INFO  State of operation 19409494 : EXECUTING
12:22:24 INFO  State of operation 19409494 : EXECUTING
12:22:26 INFO  State of operation 19409494 : EXECUTING
12:22:28 INFO  State of operation 19409494 : EXECUTING
12:22:30 INFO  State of operation 19409494 : EXECUTING
12:22:33 INFO  State of operation 19409494 : EXECUTING
12:22:36 INFO  State of operation 19409494 : EXECUTING
12:22:38 INFO  State of operation 19409494 : EXECUTING
12:22:40 INFO  State of operation 19409494 : EXECUTING
12:22:43 INFO  State of operation 19409494 : EXECUTING
12:22:45 INFO  State of operation 19409494 : EXECUTING
12:22:47 INFO  State of operation 19409494 : EXECUTING
12:22:49 INFO  State of operation 19409494 : EXECUTING
12:22:52 INFO  State of operation 19409494 : EXECUTING
12:22:54 INFO  State of operation 19409494 : EXECUTING
12:22:56 INFO  State of operation 19409494 : EXECUTING
12:22:59 INFO  State of operation 19409494 : EXECUTING
12:23:01 INFO  State of operation 19409494 : EXECUTING
12:23:03 INFO  State of operation 19409494 : EXECUTING
12:23:06 INFO  State of operation 19409494 : EXECUTING
12:23:08 INFO  State of operation 19409494 : EXECUTING
12:23:11 INFO  State of operation 19409494 : EXECUTING
12:23:13 INFO  State of operation 19409494 : EXECUTING
12:23:15 INFO  State of operation 19409494 : EXECUTING
12:23:18 INFO  State of operation 19409494 : EXECUTING
12:23:20 INFO  State of operation 19409494 : EXECUTING
12:23:22 INFO  State of operation 19409494 : EXECUTING
12:23:25 INFO  State of operation 19409494 : EXECUTING
12:23:27 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
12:23:27 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
12:23:27 INFO  
12:23:27 INFO  cleanup:
12:23:32 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:23:33 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:23:33 INFO  
12:23:33 INFO  Test duration: 177.21 secs
12:23:33 INFO  Test final outcome:  BLOCKED
12:23:33 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
12:23:33 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.02 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.02 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>12:07:45 INFO  ==============================================================
12:07:45 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
12:07:45 INFO  Dir:   software_management_end_to_end/
12:07:45 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
12:07:45 INFO  ==============================================================
12:07:46 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:07:46 INFO  EnvironmentC8y Setup
12:07:47 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:08:01 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:08:02 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:08:02 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:08:03 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:08:05 INFO  State of current operation: SUCCESSFUL
12:08:05 INFO  Response status: 201
12:08:05 INFO  Response to action: {
    "creationTime": "2022-05-20T12:08:05.888Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409375",
    "id": "19409375",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
12:08:05 INFO  Started operation: {'creationTime': '2022-05-20T12:08:05.888Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409375', 'id': '19409375', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
12:08:08 INFO  State of operation 19409375 : PENDING
12:08:10 INFO  State of operation 19409375 : EXECUTING
12:08:12 INFO  State of operation 19409375 : EXECUTING
12:08:14 INFO  State of operation 19409375 : EXECUTING
12:08:17 INFO  State of operation 19409375 : FAILED
12:08:17 ERROR Stopping as the operation has failed
12:08:17 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
12:08:17 WARN  SystemError: The operation has failed ... blocked 
12:08:17 INFO  
12:08:17 INFO  cleanup:
12:08:21 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:08:21 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:08:21 INFO  
12:08:21 INFO  Test duration: 36.50 secs
12:08:21 INFO  Test final outcome:  BLOCKED
12:08:21 INFO  Test outcome reason: SystemError: The operation has failed
12:08:21 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>12:26:10 INFO  ==============================================================
12:26:10 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
12:26:10 INFO  Dir:   software_management_end_to_end/
12:26:10 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
12:26:10 INFO  ==============================================================
12:26:10 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:26:10 INFO  EnvironmentC8y Setup
12:26:10 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:26:10 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
12:26:10 INFO  Delaying execution by 6 seconds
12:26:30 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:26:31 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:26:31 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:26:32 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:26:34 INFO  State of current operation: SUCCESSFUL
12:26:34 INFO  Response status: 201
12:26:34 INFO  Response to action: {
    "creationTime": "2022-05-20T12:26:34.609Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409542",
    "id": "19409542",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
12:26:34 INFO  Started operation: {'creationTime': '2022-05-20T12:26:34.609Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409542', 'id': '19409542', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
12:26:36 INFO  State of operation 19409542 : PENDING
12:26:39 INFO  State of operation 19409542 : EXECUTING
12:26:41 INFO  State of operation 19409542 : EXECUTING
12:26:43 INFO  State of operation 19409542 : EXECUTING
12:26:45 INFO  State of operation 19409542 : FAILED
12:26:45 ERROR Stopping as the operation has failed
12:26:45 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
12:26:45 WARN  SystemError: The operation has failed ... blocked 
12:26:45 INFO  
12:26:45 INFO  cleanup:
12:26:49 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:26:50 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:26:50 INFO  
12:26:50 INFO  Test duration: 40.37 secs
12:26:50 INFO  Test final outcome:  BLOCKED
12:26:50 INFO  Test outcome reason: SystemError: The operation has failed
12:26:50 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_fake_install_remove
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.02 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>11:24:16 INFO  ==============================================================
11:24:16 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
11:24:16 INFO  Dir:   software_management_end_to_end/
11:24:16 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
11:24:16 INFO  ==============================================================
11:24:16 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
11:24:16 WARN  Aborted test due to SKIPPED outcome
11:24:16 INFO  
11:24:16 INFO  Test duration: 0.03 secs
11:24:16 INFO  Test final outcome:  SKIPPED
11:24:16 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
11:24:16 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>11:24:17 INFO  ==============================================================
11:24:17 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
11:24:17 INFO  Dir:   software_management_end_to_end/
11:24:17 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
11:24:17 INFO  ==============================================================
11:24:17 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
11:24:17 INFO  
11:24:17 INFO  Test duration: 0.02 secs
11:24:17 INFO  Test final outcome:  SKIPPED
11:24:17 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
11:24:17 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>12:28:22 INFO  ==============================================================
12:28:22 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
12:28:22 INFO  Dir:   software_management_end_to_end/
12:28:22 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
12:28:22 INFO  ==============================================================
12:28:22 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
12:28:22 INFO  
12:28:22 INFO  Test duration: 0.03 secs
12:28:22 INFO  Test final outcome:  SKIPPED
12:28:22 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
12:28:22 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>11:24:17 INFO  ==============================================================
11:24:17 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
11:24:17 INFO  Dir:   software_management_end_to_end/
11:24:17 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
11:24:17 INFO  ==============================================================
11:24:17 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
11:24:17 WARN  Aborted test due to SKIPPED outcome
11:24:17 INFO  
11:24:17 INFO  Test duration: 0.02 secs
11:24:17 INFO  Test final outcome:  SKIPPED
11:24:17 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
11:24:17 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>12:28:22 INFO  ==============================================================
12:28:22 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
12:28:22 INFO  Dir:   software_management_end_to_end/
12:28:22 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
12:28:22 INFO  ==============================================================
12:28:22 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:28:22 INFO  EnvironmentC8y Setup
12:28:22 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:28:22 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
12:28:22 INFO  Delaying execution by 6 seconds
12:28:42 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
12:28:43 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:28:43 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:28:44 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:28:46 INFO  State of current operation: SUCCESSFUL
12:28:46 INFO  Package apple is installed
12:28:46 INFO  Assert that (True == value) with value=True ... passed
12:28:46 INFO  Package banana is installed
12:28:46 INFO  Assert that (True == value) with value=True ... passed
12:28:47 INFO  Package cherry is installed
12:28:47 INFO  Assert that (True == value) with value=True ... passed
12:28:47 INFO  Package asciijump is installed
12:28:47 WARN  Assert that (False == value) with value=True ... failed 
12:28:47 INFO  Assert that (False == value) with value=False ... passed
12:28:48 INFO  Response status: 201
12:28:48 INFO  Response to action: {
    "creationTime": "2022-05-20T12:28:47.979Z",
    "deviceId": "19396607",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409608",
    "id": "19409608",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
12:28:48 INFO  Started operation: {'creationTime': '2022-05-20T12:28:47.979Z', 'deviceId': '19396607', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409608', 'id': '19409608', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
12:28:50 INFO  State of operation 19409608 : PENDING
12:28:52 INFO  State of operation 19409608 : EXECUTING
12:28:54 INFO  State of operation 19409608 : EXECUTING
12:28:56 INFO  State of operation 19409608 : EXECUTING
12:28:59 INFO  State of operation 19409608 : EXECUTING
12:29:01 INFO  State of operation 19409608 : EXECUTING
12:29:04 INFO  State of operation 19409608 : EXECUTING
12:29:06 INFO  State of operation 19409608 : EXECUTING
12:29:09 INFO  State of operation 19409608 : EXECUTING
12:29:11 INFO  State of operation 19409608 : EXECUTING
12:29:13 INFO  State of operation 19409608 : EXECUTING
12:29:16 INFO  State of operation 19409608 : EXECUTING
12:29:18 INFO  State of operation 19409608 : EXECUTING
12:29:20 INFO  State of operation 19409608 : EXECUTING
12:29:22 INFO  State of operation 19409608 : EXECUTING
12:29:24 INFO  State of operation 19409608 : EXECUTING
12:29:27 INFO  State of operation 19409608 : FAILED
12:29:27 ERROR Stopping as the operation has failed
12:29:27 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
12:29:27 WARN  SystemError: The operation has failed ... blocked 
12:29:27 INFO  
12:29:27 INFO  cleanup:
12:29:31 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:29:31 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:29:31 INFO  
12:29:31 INFO  Test duration: 69.36 secs
12:29:31 INFO  Test final outcome:  BLOCKED
12:29:31 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
12:29:31 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session">
		<failure message="FAILED: Grep on sub_resp.out contains &quot;Ld3KgqpcLDlrYH6sfpG7w&quot;" type="FAILED" />
		<system-out>11:25:08 INFO  ==============================================================
11:25:08 INFO  Id:    agent_init_session
11:25:08 INFO  Dir:   tedge_agent/
11:25:08 INFO  Title: Validate tedge agent init session feature
11:25:08 INFO  ==============================================================
11:25:09 INFO  Executed sudo&lt;remove_lock&gt;, exit status 0
11:25:09 INFO  Executed sudo&lt;agent_clear&gt;, exit status 0
11:25:10 INFO  Executed sudo&lt;agent_init&gt;, exit status 0
11:25:11 INFO  Executed sudo&lt;pub_req&gt;, exit status 0
11:25:11 INFO  Started sudo&lt;sub_resp&gt; with process id 2986
11:25:11 INFO  Executed sudo&lt;agent_start&gt;, exit status 0
11:25:11 INFO  Waiting for 2.0 seconds
11:25:14 INFO  Executed sudo&lt;agent_stop&gt;, exit status 0
11:25:14 INFO  Executed sudo&lt;sub_stop&gt;, exit status 0
11:25:14 WARN  Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w" ... failed [run.py:87]
11:25:14 INFO  
11:25:14 INFO  Test duration: 5.90 secs
11:25:14 INFO  Test final outcome:  FAILED
11:25:14 INFO  Test outcome reason: Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w"
11:25:14 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    apt_fail_noarg
11:14:38 INFO  Dir:   plugin_apt/
11:14:38 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
11:14:38 INFO  ==============================================================
11:14:38 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.04 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    apt_finalize
11:14:38 INFO  Dir:   plugin_apt/
11:14:38 INFO  Title: Validate apt plugin finalize - dense version
11:14:38 INFO  ==============================================================
11:14:38 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.02 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    apt_install
11:14:38 INFO  Dir:   plugin_apt/
11:14:38 INFO  Title: Validate apt plugin prepare - dense version
11:14:38 INFO  ==============================================================
11:14:38 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.02 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:38 INFO  ==============================================================
11:14:38 INFO  Id:    apt_install_fail
11:14:38 INFO  Dir:   plugin_apt/
11:14:38 INFO  Title: Validate apt plugin prepare - failing
11:14:38 INFO  ==============================================================
11:14:38 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:38 WARN  Aborted test due to SKIPPED outcome
11:14:38 INFO  
11:14:38 INFO  Test duration: 0.02 secs
11:14:38 INFO  Test final outcome:  SKIPPED
11:14:38 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:38 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:39 INFO  ==============================================================
11:14:39 INFO  Id:    apt_install_local_file
11:14:39 INFO  Dir:   plugin_apt/
11:14:39 INFO  Title: Validate apt plugin install from local file - SUCCESS case
11:14:39 INFO  ==============================================================
11:14:39 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:39 WARN  Aborted test due to SKIPPED outcome
11:14:39 INFO  
11:14:39 INFO  Test duration: 0.02 secs
11:14:39 INFO  Test final outcome:  SKIPPED
11:14:39 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:39 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:39 INFO  ==============================================================
11:14:39 INFO  Id:    apt_install_local_file_fail
11:14:39 INFO  Dir:   plugin_apt/
11:14:39 INFO  Title: Validate apt plugin install from local file - FAIL case
11:14:39 INFO  ==============================================================
11:14:39 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:39 WARN  Aborted test due to SKIPPED outcome
11:14:39 INFO  
11:14:39 INFO  Test duration: 0.03 secs
11:14:39 INFO  Test final outcome:  SKIPPED
11:14:39 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:39 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:39 INFO  ==============================================================
11:14:39 INFO  Id:    apt_install_local_file_with_version
11:14:39 INFO  Dir:   plugin_apt/
11:14:39 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
11:14:39 INFO  ==============================================================
11:14:39 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:39 WARN  Aborted test due to SKIPPED outcome
11:14:39 INFO  
11:14:39 INFO  Test duration: 0.02 secs
11:14:39 INFO  Test final outcome:  SKIPPED
11:14:39 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:39 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:39 INFO  ==============================================================
11:14:39 INFO  Id:    apt_install_local_file_with_version_fail
11:14:39 INFO  Dir:   plugin_apt/
11:14:39 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
11:14:39 INFO  ==============================================================
11:14:39 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:39 WARN  Aborted test due to SKIPPED outcome
11:14:39 INFO  
11:14:39 INFO  Test duration: 0.02 secs
11:14:39 INFO  Test final outcome:  SKIPPED
11:14:39 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:39 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_prepare
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin install - dense version
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_prepare_install_finalize
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin install use case
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_remove
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin prepare - dense version
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_remove_fail
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin prepare - failing
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_sequence_install_with_version
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin install with version use case
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.03 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_sequence_install_with_version_fail
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin install with version fails use case
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_sequence_remove
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin prepare install remove finalize use case
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_sequence_remove_with_version
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin remove with version use case
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_sequence_remove_with_version_fail
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin remove with version fails when the version not found
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:42 INFO  ==============================================================
11:14:42 INFO  Id:    apt_update_list
11:14:42 INFO  Dir:   plugin_apt/
11:14:42 INFO  Title: Validate apt plugin update-list command use case
11:14:42 INFO  ==============================================================
11:14:42 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:42 WARN  Aborted test due to SKIPPED outcome
11:14:42 INFO  
11:14:42 INFO  Test duration: 0.02 secs
11:14:42 INFO  Test final outcome:  SKIPPED
11:14:42 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:42 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:14:43 INFO  ==============================================================
11:14:43 INFO  Id:    apt_wrong_option
11:14:43 INFO  Dir:   plugin_apt/
11:14:43 INFO  Title: Validate apt plugin install
11:14:43 INFO  ==============================================================
11:14:43 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:14:43 WARN  Aborted test due to SKIPPED outcome
11:14:43 INFO  
11:14:43 INFO  Test duration: 0.02 secs
11:14:43 INFO  Test final outcome:  SKIPPED
11:14:43 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:14:43 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval" />
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V" />
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:14:43 INFO  ==============================================================
11:14:43 INFO  Id:    docker_plugin_finalize
11:14:43 INFO  Dir:   plugin_docker/
11:14:43 INFO  Title: Validate docker plugin finalize command
11:14:43 INFO  ==============================================================
11:14:43 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:14:43 WARN  Aborted test due to SKIPPED outcome
11:14:43 INFO  
11:14:43 INFO  Test duration: 0.05 secs
11:14:43 INFO  Test final outcome:  SKIPPED
11:14:43 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:14:43 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize" />
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:14:43 INFO  ==============================================================
11:14:43 INFO  Id:    docker_plugin_install
11:14:43 INFO  Dir:   plugin_docker/
11:14:43 INFO  Title: Validate docker plugin install
11:14:43 INFO  ==============================================================
11:14:43 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:14:43 WARN  Aborted test due to SKIPPED outcome
11:14:43 INFO  
11:14:43 INFO  Test duration: 0.02 secs
11:14:43 INFO  Test final outcome:  SKIPPED
11:14:43 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:14:43 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install" />
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:14:43 INFO  ==============================================================
11:14:43 INFO  Id:    docker_plugin_install_with_version
11:14:43 INFO  Dir:   plugin_docker/
11:14:43 INFO  Title: Validate docker plugin install
11:14:43 INFO  ==============================================================
11:14:43 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:14:43 WARN  Aborted test due to SKIPPED outcome
11:14:43 INFO  
11:14:43 INFO  Test duration: 0.02 secs
11:14:43 INFO  Test final outcome:  SKIPPED
11:14:43 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:14:43 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>12:07:24 INFO  ==============================================================
12:07:24 INFO  Id:    docker_plugin_install_with_version
12:07:24 INFO  Dir:   plugin_docker/
12:07:24 INFO  Title: Validate docker plugin install
12:07:24 INFO  ==============================================================
12:07:25 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
12:07:25 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out ... passed
12:07:27 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
12:07:27 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:24]
12:07:28 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
12:07:29 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
12:07:29 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:28]
12:07:29 INFO  
12:07:29 INFO  cleanup:
12:07:29 INFO  Removing containers scheduled to be cleaned up: ['420912de889d', '']
12:07:30 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
12:07:30 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
12:07:31 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
12:07:31 INFO  
12:07:31 INFO  Test duration: 6.88 secs
12:07:31 INFO  Test final outcome:  BLOCKED
12:07:31 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
12:07:31 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:14:43 INFO  ==============================================================
11:14:43 INFO  Id:    docker_plugin_remove
11:14:43 INFO  Dir:   plugin_docker/
11:14:43 INFO  Title: Validate docker plugin remove
11:14:43 INFO  ==============================================================
11:14:43 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:14:43 WARN  Aborted test due to SKIPPED outcome
11:14:43 INFO  
11:14:43 INFO  Test duration: 0.02 secs
11:14:43 INFO  Test final outcome:  SKIPPED
11:14:43 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:14:43 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>12:07:31 INFO  ==============================================================
12:07:31 INFO  Id:    docker_plugin_remove
12:07:31 INFO  Dir:   plugin_docker/
12:07:31 INFO  Title: Validate docker plugin remove
12:07:31 INFO  ==============================================================
12:07:32 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
12:07:32 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out ... passed
12:07:34 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
12:07:34 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:latest' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

12:07:34 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
12:07:34 WARN  Aborted test due to BLOCKED outcome
12:07:34 INFO  
12:07:34 INFO  cleanup:
12:07:34 INFO  Removing containers scheduled to be cleaned up: ['420912de889d', '']
12:07:34 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
12:07:34 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
12:07:35 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
12:07:35 INFO  
12:07:35 INFO  Test duration: 4.61 secs
12:07:35 INFO  Test final outcome:  BLOCKED
12:07:35 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
12:07:35 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:14:43 INFO  ==============================================================
11:14:43 INFO  Id:    docker_plugin_remove_with_version
11:14:43 INFO  Dir:   plugin_docker/
11:14:43 INFO  Title: Validate docker plugin remove with version
11:14:43 INFO  ==============================================================
11:14:43 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:14:43 WARN  Aborted test due to SKIPPED outcome
11:14:43 INFO  
11:14:43 INFO  Test duration: 0.02 secs
11:14:43 INFO  Test final outcome:  SKIPPED
11:14:43 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:14:43 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>12:07:35 INFO  ==============================================================
12:07:35 INFO  Id:    docker_plugin_remove_with_version
12:07:35 INFO  Dir:   plugin_docker/
12:07:35 INFO  Title: Validate docker plugin remove with version
12:07:35 INFO  ==============================================================
12:07:36 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
12:07:36 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove_with_version/Output/linux/docker_images.out ... passed
12:07:38 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
12:07:38 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.7.1' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

12:07:38 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
12:07:38 WARN  Aborted test due to BLOCKED outcome
12:07:38 INFO  
12:07:38 INFO  cleanup:
12:07:38 INFO  Removing containers scheduled to be cleaned up: ['420912de889d', '']
12:07:39 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
12:07:39 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
12:07:40 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
12:07:40 INFO  
12:07:40 INFO  Test duration: 4.62 secs
12:07:40 INFO  Test final outcome:  BLOCKED
12:07:40 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
12:07:40 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:14:43 INFO  ==============================================================
11:14:43 INFO  Id:    docker_plugin_update
11:14:43 INFO  Dir:   plugin_docker/
11:14:43 INFO  Title: Validate docker plugin install updates existing containers
11:14:43 INFO  ==============================================================
11:14:43 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:14:43 WARN  Aborted test due to SKIPPED outcome
11:14:43 INFO  
11:14:43 INFO  Test duration: 0.02 secs
11:14:43 INFO  Test final outcome:  SKIPPED
11:14:43 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:14:43 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>12:07:40 INFO  ==============================================================
12:07:40 INFO  Id:    docker_plugin_update
12:07:40 INFO  Dir:   plugin_docker/
12:07:40 INFO  Title: Validate docker plugin install updates existing containers
12:07:40 INFO  ==============================================================
12:07:41 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
12:07:41 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
12:07:43 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
12:07:43 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.6.2' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

12:07:43 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
12:07:43 WARN  Aborted test due to BLOCKED outcome
12:07:43 INFO  
12:07:43 INFO  cleanup:
12:07:43 INFO  Removing containers scheduled to be cleaned up: ['420912de889d', '']
12:07:44 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
12:07:44 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
12:07:45 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
12:07:45 INFO  
12:07:45 INFO  Test duration: 4.63 secs
12:07:45 INFO  Test final outcome:  BLOCKED
12:07:45 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
12:07:45 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_fail_noqos" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_noqos" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos0" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos0" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos1" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos1" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos2" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos2" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain" />
</testsuite><testsuite name="local_mqtt_pub_sub_with_sudo" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub_with_sudo" />
</testsuite><testsuite name="local_mqtt_sub_fail" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_sub_fail" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>11:11:39 INFO  ==============================================================
11:11:39 INFO  Id:    log_request_error_messages
11:11:39 INFO  Dir:   log_request_end_to_end/
11:11:39 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
11:11:39 INFO  ==============================================================
11:11:40 INFO  EnvironmentC8y Setup
11:11:41 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:11:41 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:11:41 INFO  Delaying execution by 8 seconds
11:12:03 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:12:04 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:12:04 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:12:05 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:12:05 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
11:12:06 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
11:12:07 INFO  log response None
11:12:07 WARN  retry, retrying in 1 seconds...
11:12:08 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/19395849/binaries
11:12:09 WARN  Assert that (True == value) with value=False ... failed [run.py:43]
11:12:09 INFO  
11:12:09 INFO  cleanup:
11:12:09 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
11:12:10 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
11:12:10 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
11:12:14 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:12:15 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:12:15 INFO  
11:12:15 INFO  Test duration: 35.86 secs
11:12:15 INFO  Test final outcome:  FAILED
11:12:15 INFO  Test outcome reason: Assert that (True == value) with value=False
11:12:15 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>11:12:15 INFO  ==============================================================
11:12:15 INFO  Id:    log_request_num_lines
11:12:15 INFO  Dir:   log_request_end_to_end/
11:12:15 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
11:12:15 INFO  ==============================================================
11:12:15 INFO  EnvironmentC8y Setup
11:12:16 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:12:16 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
11:12:16 INFO  Delaying execution by 7 seconds
11:12:37 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:12:38 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:12:38 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:12:39 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:12:39 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
11:12:40 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
11:12:40 INFO  log response None
11:12:40 WARN  retry, retrying in 1 seconds...
11:12:42 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/19398704/binaries
11:12:42 INFO  nlines 0
11:12:42 WARN  Assert that (True == value) with value=False ... failed [run.py:44]
11:12:42 INFO  
11:12:42 INFO  cleanup:
11:12:42 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
11:12:43 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
11:12:44 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
11:12:48 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:12:48 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:12:48 INFO  
11:12:48 INFO  Test duration: 33.38 secs
11:12:48 INFO  Test final outcome:  FAILED
11:12:48 INFO  Test outcome reason: Assert that (True == value) with value=False
11:12:48 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>11:27:19 INFO  ==============================================================
11:27:19 INFO  Id:    mapper_awaits_before_reconnect
11:27:19 INFO  Dir:   tedge_mapper_c8y/
11:27:19 INFO  Title: Validate mapper doesn't reconnect too often
11:27:19 INFO  ==============================================================
11:27:19 INFO  Started mosquitto_sub with process id 3745
11:27:19 INFO  Waiting for 0.1 seconds
11:27:19 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
11:27:19 INFO  Waiting for 5.0 seconds
11:27:25 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
11:27:26 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
11:27:26 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
11:27:26 INFO  Line count on file tedge_mapper_journal.out ... passed
11:27:26 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
11:27:26 WARN  Aborted test due to FAILED outcome
11:27:26 INFO  
11:27:26 INFO  Test duration: 7.92 secs
11:27:26 INFO  Test final outcome:  FAILED
11:27:26 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
11:27:26 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>11:14:18 INFO  ==============================================================
11:14:18 INFO  Id:    monitoring_smaller_interval
11:14:18 INFO  Dir:   monitoring/
11:14:18 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
11:14:18 INFO  ==============================================================
11:14:19 INFO  Executed sudo&lt;collectd&gt;, exit status 0
11:14:20 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:14:20 INFO  Started sudo&lt;tedge_sub&gt; with process id 32339
11:14:20 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:21 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:21 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:22 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:23 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:23 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:24 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:24 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:25 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:25 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:26 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:27 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:27 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:28 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:28 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:29 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:29 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:30 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:31 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:31 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:33 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
11:14:33 INFO  0

11:14:33 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
11:14:33 WARN  TypeError: 'int' object is not subscriptable ... blocked 
11:14:33 INFO  
11:14:33 INFO  cleanup:
11:14:33 INFO  monitoring_cleanup
11:14:33 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:14:33 INFO  
11:14:33 INFO  Test duration: 15.47 secs
11:14:33 INFO  Test final outcome:  BLOCKED
11:14:33 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
11:14:33 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>11:14:33 INFO  ==============================================================
11:14:33 INFO  Id:    monitoring_with_collectd
11:14:33 INFO  Dir:   monitoring/
11:14:33 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
11:14:33 INFO  ==============================================================
11:14:33 INFO  Test execution disabled due to #792 (instable) ... skipped
11:14:33 INFO  
11:14:33 INFO  Test duration: 0.02 secs
11:14:33 INFO  Test final outcome:  SKIPPED
11:14:33 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
11:14:33 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>11:14:33 INFO  ==============================================================
11:14:33 INFO  Id:    monitoring_with_simulated_messages
11:14:33 INFO  Dir:   monitoring/
11:14:33 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
11:14:33 INFO  ==============================================================
11:14:34 INFO  Executed sudo&lt;collectd&gt;, exit status 0
11:14:35 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:14:35 INFO  Started sudo&lt;tedge_sub&gt; with process id 32390
11:14:35 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:14:36 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:14:37 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
11:14:37 INFO  0

11:14:37 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
11:14:37 WARN  TypeError: 'int' object is not subscriptable ... blocked 
11:14:37 INFO  
11:14:37 INFO  cleanup:
11:14:37 INFO  monitoring_cleanup
11:14:38 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:14:38 INFO  
11:14:38 INFO  Test duration: 4.43 secs
11:14:38 INFO  Test final outcome:  BLOCKED
11:14:38 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
11:14:38 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>11:13:23 INFO  ==============================================================
11:13:23 INFO  Id:    mqtt_port_change_connection_works
11:13:23 INFO  Dir:   misc_features/
11:13:23 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
11:13:23 INFO  ==============================================================
11:13:23 INFO  failing intermitently ... skipped
11:13:23 INFO  
11:13:23 INFO  Test duration: 0.02 secs
11:13:23 INFO  Test final outcome:  SKIPPED
11:13:23 INFO  Test outcome reason: failing intermitently
11:13:23 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_set" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortSet" name="mqtt_port_set" />
</testsuite><testsuite name="mqtt_port_unset" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortUnSet" name="mqtt_port_unset" />
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers" />
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json" />
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms" />
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms" />
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms" />
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>11:24:42 INFO  ==============================================================
11:24:42 INFO  Id:    tedge_agent_user_sudo_access
11:24:42 INFO  Dir:   tedge/
11:24:42 INFO  Title: Validate tedge-agent user has a limited sudo right
11:24:42 INFO  ==============================================================
11:24:42 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
11:24:42 INFO  
11:24:42 INFO  Test duration: 0.02 secs
11:24:42 INFO  Test final outcome:  SKIPPED
11:24:42 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
11:24:42 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms" />
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement" />
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>11:29:03 INFO  ==============================================================
11:29:03 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
11:29:03 INFO  Dir:   tedge_mapper_c8y/
11:29:03 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
11:29:03 INFO  ==============================================================
11:29:04 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
11:29:04 INFO  Started sudo&lt;tedge_sub&gt; with process id 4343
11:29:04 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
11:29:05 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
11:29:05 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
11:29:05 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
11:29:05 INFO  
11:29:05 INFO  cleanup:
11:29:05 INFO  mapper_cleanup
11:29:06 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
11:29:06 INFO  
11:29:06 INFO  Test duration: 3.01 secs
11:29:06 INFO  Test final outcome:  FAILED
11:29:06 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
11:29:06 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>